src/hexen/h2_main.c

```diff
@@ -676,6 +676,23 @@ static void HandleArgs(void)
         ST_Message("Playing demo %s.\n", myargv[p+1]);
     }
 
+    //!
+    // @category demo
+    //
+    // Record or playback a demo with high resolution turning.
```

Comment:
Sounds great! I'm not familiar with the full feature set for Heretic+ and Hexen+ (and supporting all of it should be out of scope for Chocolate Doom) but compatibility with the demo format at least seems in line with the Doom 1.91 longtics demo format that we already support.